resource_detectors/opentelemetry-resource_detectors.gemspec

```diff
@@ -26,7 +26,8 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.6.0'
 
   spec.add_dependency 'google-cloud-env'
-  spec.add_dependency 'opentelemetry-sdk'
+  spec.add_dependency 'opentelemetry-registry', '~> 0.1'
```

Comment:
This happens to be because the SDK has not yet been released with the transitive dependency on the registry _and_ the resource detectors is depending on a locally installed version of the SDK that _does_.

That means that the resource detectors will not work locally until a version of the SDK is released with the registry dependency. 

Maybe a smaller step I can take here is releasing a version of the SDK that depends on the registry and then a follow up PR to update everything else. 🤷🏼

Reply:
fixed 525bc1e5e366a01d3798a1f66ff0292c4ce060e4